### PR TITLE
Features/admin process queue task grid

### PR DESCRIPTION
**Pull request by dunagan5887**
Providing a grid showing all order creation queue tasks

### CODE REVIEW

**Comment by dunagan5887** [comment]
Actually just thought of something: we should return any curl adapter errors to be seen in the admin panel. This shouldn't take me long to include here

**Comment by skwp** [comment]
That would be very useful yes. especially with customers who don't know how to get at logs

**Comment by skwp** [comment]
Can I still merge this?

File: app/code/community/Reverb/Base/Block/Adminhtml/Widget/Form.php [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    ),
+    array(
+    'label' => 'label_for_this_option',
+    'value' =>  'value_for_this_option'
+    ),
+    ...
+    ...
+    ...
+    );
+     * @param bool $required - Defaults to true
```
Comment (skwp) [inline app/code/community/Reverb/Base/Block/Adminhtml/Widget/Form.php]: weird indent?

**Comment by dunagan5887** [comment]
I'll have the Curl error messaging update in about 15 minutes if you want to hold on for that

File: app/code/community/Reverb/Base/Block/Adminhtml/Widget/Form.php (hunk 1 above)
Comment (dunagan5887) [inline app/code/community/Reverb/Base/Block/Adminhtml/Widget/Form.php]: I'll remove that comment

File: app/code/community/Reverb/ProcessQueue/Model/Source/Task/Status.php [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                    'label' => Mage::helper('reverb_process_queue')->__('Complete'),
+                    'value' => Reverb_ProcessQueue_Model_Task::STATUS_COMPLETE
+                ),
+                array(
+                    'label' => Mage::helper('reverb_process_queue')->__('Error'),
+                    'value' => Reverb_ProcessQueue_Model_Task::STATUS_ERROR
+                ),
+                array(
+                    'label' => Mage::helper('reverb_process_queue')->__('Aborted'),
+                    'value' => Reverb_ProcessQueue_Model_Task::STATUS_ABORTED
```
Comment (skwp) [inline app/code/community/Reverb/ProcessQueue/Model/Source/Task/Status.php]: does aborted mean aborted by user? let's be clear about what these mean

Comment (dunagan5887) [inline app/code/community/Reverb/ProcessQueue/Model/Source/Task/Status.php]: Aborted means it should not be re-attempted e.g. an order was already synced from Reverb

Comment (dunagan5887) [inline app/code/community/Reverb/ProcessQueue/Model/Source/Task/Status.php]: If this seems unclear let me know what labels you would like the admins to see

**Comment by dunagan5887** [comment]
Ok the code looks good, preparing the commit now

**Comment by dunagan5887** [comment]
CURL errors will now be shown in the admin panel

**Comment by dunagan5887** [comment]
Going to do a quick test of the module on a clean Magento instance

**Comment by dunagan5887** [comment]
Ok at this point the module looks ready for a new release for Drum Center. The new logging should make it much easier to debug issues

**Comment by skwp** [comment]
thanks for the heads up

**Pull request merged**